src/commodore-helm/index.ts

```diff
@@ -94,7 +96,7 @@ export function extractPackageFile(
         if (d && d.depName) {
           res.depName = d.depName;
         } else {
-          res.skipReason = SkipReason.InvalidDependencySpecification;
+          res.skipReason = 'invalid-dependency-specification';
```

Comment:
Did upstream really decide that hardcoded values are a better way to do this? :see_no_evil:

Reply:
They now use a sum type over a bunch of string literals, cf. https://github.com/renovatebot/renovate/blob/a241b03c93906ace657d99c73506830a62d08c29/lib/types/skip-reason.ts

Reply:
Thank's. I hate it.